install.md

```diff
@@ -95,6 +95,24 @@ now?</q>. Confirm this by clicking <q>Install</q>.</li>
 contains executables such as <code>clang</code> and <code>clang++</code>,
 the C and C++ compiler.</li>
 </ol>
+<p>
+Note: some users have experienced build failures on macOS 10.15 with Xcode 11.4
+(cf. this <a href="https://github.com/oscar-system/Oscar.jl/issues/82">github issue</a>).
+Until a proper fix is found, the following might serve as a workaround, consisting in building <code>libcxxwrap-julia</code>
+with Xcode 11.4 instead of using the binaries (replace <code>$SOMEWHERE</code> by
+the directory in which you wish to install <code>libcxxwrap-julia</code>):
+{% highlight bash %}
+git clone -b v0.6.6 --depth 1 https://github.com/JuliaInterop/libcxxwrap-julia
```

Comment:
So this is specific to CxxWrap 0.9 (in 0.10, version 0.7 of libcxxwrap is used). We will have to update it when we switch to 0.10 then.

Reply:
Yeah, I don't know what else to do here.

Reply:
This is fine -- I meant this more as a reminder: "once we update the packages, we need to revisit this (and e.g. ask Benjamin for help with an updated text). Of course in an ideal world, the issue would simply be fixed / workaround by the CxxWrap author :-)